packages/preview/lab-report-template-ustc/1.0.0/content/example/main.typ

```diff
@@ -0,0 +1,477 @@
+#import "../../theme.typ": *
```

Comment:
This import should be absolute (i.e `#import "@preview/...": *`).

Reply:
sorry, I forgot to change it.